views/home.js

```diff
@@ -60,51 +73,61 @@ const Home = (props) => {
       }
     } else {
       // Box is empty if user is not logged in.
-      return '';
+      return "";
     }
   };
 
   return (
     <>
-      <PageHead/>
+      <PageHead />
       <div className="page_container">
         <MusaicHeader
           loggedIn={props.loggedIn}
           loginFunction={props.loginFunction}
           logoutFunction={props.logoutFunction}
         />
+        {props.error !== "" ? (
+          <Alert
+            variant="primary"
+            className="justify-content-md-center w-50 mx-auto mb-0"
+          >
+            {props.error}
+          </Alert>
+        ) : (
+          ""
+        )}
         <div className={styles.big_flex_container}>
           <div className={`${styles.row} ${styles.row_1}`}>
             <div className={`${styles.intro_box} ${styles.box}`}>
               {props.username ? `Welcome, ${props.username}!` : "Welcome!"}
             </div>
             <div className={`${styles.instructions_box} ${styles.box}`}>
-              <p><b>Here's how this works:</b> Answer a few questions, and we'll make you a brand new Spotify playlist
-              based on your mood and musical taste! You'll also get a mood, which you can reuse to
-              create more playlists from your responses. Or, share your moods with friends so they can create their
-              own playlists from your musical mood!</p>
+              <p>
+                <b>Here's how this works:</b> Answer a few questions, and we'll
+                make you a brand new Spotify playlist based on your mood and
+                musical taste! You'll also get a mood, which you can reuse to
+                create more playlists from your responses. Or, share your moods
+                with friends so they can create their own playlists from your
+                musical mood!
```

Comment:
Not sure if we're still going to go ahead with the sharing with friends functionality

Reply:
Oh yeah, we should change that